src/CodeViewer/Viewer.cpp

```diff
@@ -126,8 +124,16 @@ void Viewer::resizeEvent(QResizeEvent* ev) {
 }
 
 void Viewer::wheelEvent(QWheelEvent* ev) {
+  auto document_default_font = document()->defaultFont();
```

Comment:
Should this (and below) be an explicit type? https://google.github.io/styleguide/cppguide.html#Type_deduction
(Disclaimer: I personally like auto for cases like this, but I was corrected about this in multiple PRs now :D)

Reply:
You're probably right. I will fix it. :-)

Reply:
Done.